tests/radv/test_radv_ipv6_ra.py

```diff
@@ -4,7 +4,12 @@
 
 import pytest
 
-from tests.common.fixtures.ptfhost_utils import copy_ptftests_directory   # lgtm[py/unused-import]
+from tests.common.fixtures.ptfhost_utils import copy_ptftests_directory # lgtm[py/unused-import]
+from tests.common.fixtures.ptfhost_utils import change_mac_addresses # lgtm[py/unused-import]
+from tests.common.fixtures.ptfhost_utils import run_garp_service # lgtm[py/unused-import]
+from tests.common.fixtures.ptfhost_utils import run_icmp_responder # lgtm[py/unused-import]
+from tests.common.dualtor.dual_tor_mock import mock_server_base_ip_addr # lgtm[py/unused-import]
+from tests.common.dualtor.mux_simulator_control import toggle_all_simulator_ports_to_upper_tor
```

Comment:
Suggest using `toggle_all_simulator_ports_to_rand_selected_tor` as per https://github.com/Azure/sonic-mgmt/pull/4188/files#r703779335

Reply:
i have now removed if/else

Reply:
> Suggest using `toggle_all_simulator_ports_to_rand_selected_tor` as per https://github.com/Azure/sonic-mgmt/pull/4188/files#r703779335
@theasianpianist I am seeing below error when using toggle_all_simulator_ports_to_rand_selected_tor 

06:35:54 mux_simulator_control.toggle_all_simulat L0297 INFO | Toggling mux cable to str2-7050cx3-acs-09
06:36:00 mux_simulator_control._check_all_active L0319 WARNING| Unexpected mux status Ethernet0 standby unhealthy
06:36:07 mux_simulator_control._check_all_active L0319 WARNING| Unexpected mux status Ethernet0 standby unhealthy
06:36:13 mux_simulator_control._check_all_active L0319 WARNING| Unexpected mux status Ethernet0 standby unhealthy
FAILED

So, for now I am going ahead with this PR merge with always selecting 'upper_tor' and come to your suggested approach later